timerSpec.js

```diff
@@ -0,0 +1,21 @@
+describe(function(){
```

Comment:
I think test should be named `timer.test.js` and it should go in a `__tests__` directory